registration/register.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+  <link rel="stylesheet" type="text/css" href="frontend/main.css">
+  <title>Registration For SPECTRA</title>
+</head>
+<body>
+  <iframe id="typeform-full" width="100%" height="100%" frameborder="0" src="https://ingochris.typeform.com/to/gltwlZ"></iframe>
```

Comment:
> Are we able to put the typeform underneath a spectra account instead of your account?

TF won't let me transfer the form without starting over from scratch. :sob: It shouldn't show up on the page because I've embedded it in an iframe, so it'll be under https://spectrahackathon.com/register (We *are* using SSL, right?)

> Form looks awesome btw! :)

:smile_cat: 

> Please sync with marketing team about using airtable vs. typeform.. I want everyone on the same page about this before we merge your PR.

This is still in discussion (with Ashi/Cynthia/Izzy), but we might end up sticking with TF.